structuredheadings/styles/numbering.css

```diff
@@ -2,33 +2,48 @@
 body {
     counter-reset: h1 h2 h3 h4 h5 h6;
 }
-h1.autonumber {
+.autonumber.autonumber-0 {
     counter-increment: h1;
     counter-reset: h2;
 }
-h2.autonumber {
+.autonumber.autonumber-1 {
 	counter-increment: h2;
 	counter-reset: h3;
 }
-h3.autonumber {
+.autonumber.autonumber-2 {
 	counter-increment: h3;
 	counter-reset: h4;
 }
-h4.autonumber {
+.autonumber.autonumber-3 {
 	counter-increment: h4;
 	counter-reset: h5;
 }
-h5.autonumber {
+.autonumber.autonumber-4 {
 	counter-increment: h5;
 	counter-reset: h6;
 }
-h6.autonumber {
+.autonumber.autonumber-5 {
```

Comment:
Depends on where your mind is at, I've been in the mindset of "array index" which is what those classes indicate, so if I change the class to be 1-6, I'll need to adjust the code to be index + 1 :)